hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -16,14 +16,14 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.operator;
+package org.apache.hudi.configuration;
 
 import org.apache.hudi.common.model.HoodieTableType;
-import org.apache.hudi.config.HoodieWriteConfig;
-import org.apache.hudi.streamer.FlinkStreamerConfig;
 import org.apache.hudi.common.model.OverwriteWithLatestAvroPayload;
+import org.apache.hudi.config.HoodieWriteConfig;
 import org.apache.hudi.keygen.SimpleAvroKeyGenerator;
 import org.apache.hudi.keygen.constant.KeyGeneratorOptions;
+import org.apache.hudi.streamer.FlinkStreamerConfig;
```

Comment:
We have a `configuration` subpackage, can we put `FlinkStreamerConfig ` into it?

Reply:
`FlinkStreamerConfig` is only used for streamer, keep it under `streamer` package is more reasonable.